modules/apps/asset/asset-info-impl/src/main/java/com/liferay/asset/info/internal/item/provider/AssetEntryInfoItemFieldSetProviderImpl.java

```diff
@@ -115,37 +164,53 @@ public InfoFieldSet getInfoFieldSet(String itemClassName) {
 		}
 	}
 
-	private String _getCategoryNames(AssetEntry assetEntry) {
-		Locale locale = LocaleThreadLocal.getThemeDisplayLocale();
+	private Set<AssetVocabulary> _getAssetVocabularies(AssetEntry assetEntry) {
+		List<AssetCategory> assetCategories = assetEntry.getCategories();
```

Comment:
Doesn't assetEntry.getCategories() just return the currently assigned categories?

If so, this would miss all other vocabularies associated to the content type but with no category assigned.